frontend/src/features/admin-form/create/payment/PaymentDrawer.tsx

```diff
@@ -148,24 +149,28 @@ export const PaymentInput = (): JSX.Element => {
           )
         },
         validateMin: (val) => {
+          if (minPaymentAmountCents === undefined) return true
           return (
-            Number(val?.trim()) >= minPaymentAmount ||
+            // val is in dollars
+            Number(val?.trim()) >= minPaymentAmountCents / 100 ||
             `Please enter a payment amount above ${formatCurrency(
-              minPaymentAmount,
+              minPaymentAmountCents / 100,
```

Comment:
hm, maybe it will be better if we use the centToDollars function instead of dividing by 100 to avoid division errors?

Reply:
for this, i'm thinking to keep the current implementation, because:

1) (I'm assuming we would do the same for L155 too) `centsToDollars` takes a number and returns a string, so we need to cast to Number again for the comparison, which is not ideal
2) `formatCurrency` already handles the formatting for us
3) there is no division error because the denominator is > 0

thoughts?

Reply:
I'm inclined to say that (1) isn't that big of a point since we're doing the same for the input anyway, and it will not introduce any errors.
For 3, I recall there were specific numbers that became long floats then divided by 100 🤔 Let me try to find these numbers again. But yes, you're right that it will be handled by `formatCurrency` (2). 
I was discussing this with @timotheeg previously and we went with the `centsToDollars` and `dollarsToCents` implementation when converting between the values where we use string parsing for conversion as it is cleaner and avoids all the numerical errors that come with division/ multiplication. Referring to [this PR](https://github.com/opengovsg/FormSG/pull/5852), but we had the discussion face-to-face so sadly it's not documented.

Oh also to clarify, my main issue is the `/ 100` part, not the use of `formatCurrency` to string format.

Reply:
This quirk might not be significant considering there's the IRAS limit, but just something to take note in the future if the amount increases? Also the correctness of the way we process the amounts, I guess.

Reply:
Thanks for this! I can go along [e5d988d](https://github.com/opengovsg/FormSG/pull/5972/commits/e5d988d64301c0d3a9e4dd863734faa3842bf4bc)